### PR TITLE
additional docs fixes

### PR DESCRIPTION
**Pull request by yedpodtrzitko**
Hi,
this PR contains a bunch of boring changes (see below). However there's a semantic change which needs your attention, just to be sure it's correct. I renamed `NetworkObject` into `NetworkBehavior` in a few locations of the docs (dropped comments at the specific lines below).

- fix some typos
- fix/add more links across documentation
- remove orphaned redundant files

### CODE REVIEW

File: docs/mkdocs/docs/NetworkObject/RemoteProcedureCalls/rpc-validation-by-server.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 
-When we use the Network Contract Wizard (NCW), it will create 2 generated classes; one for our MonoBehaviour and one for our NetworkObject.
+When we use the Network Contract Wizard (NCW), it will create 2 generated classes; one for our `MonoBehaviour` and one for our `NetworkObject`.
 
-The one that we want to focus on in this example is the generated NetworkObject. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Ball** ; this will generate a NetworkObject class named **BallNetworkObject**. Now in another folder (not in the Generated folder) you will create a new C# script called **BallNetworkObject**. In here you will create your [partial class](https://msdn.microsoft.com/en-us/library/wa80x488.aspx) for the **BallNetworkObj ect** and you will override the **ServerAllowRpc** method like so:
+The one that we want to focus on in this example is the generated `NetworkBehavior`. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Ball** ; this will generate a `NetworkObject` class named **BallNetworkObject**.
```
Comment (yedpodtrzitko) [inline docs/mkdocs/docs/NetworkObject/RemoteProcedureCalls/rpc-validation-by-server.md]: Renamed `NetworkObject` here to `NetworkBehavior` here, and in the code example below.

File: docs/mkdocs/docs/NetworkObject/changing-ownership.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
 
 When we use the Network Contract Wizard (NCW), it will create 2 generated classes; one for our MonoBehaviour and one for our NetworkObject.
 
-The one that we want to focus on in this example is the generated NetworkObject. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Car** ; this will generate a NetworkObject class named **CarNetworkObject**. Now in another folder (not in the Generated folder) you will create a new C# script called **CarNetworkObject**. In here you will create your [partial class](https://msdn.microsoft.com/en-us/library/wa80x488.aspx) for the **CarNetworkObject** and you will override the **AllowOwnershipChange** method like so:
+The one that we want to focus on in this example is the generated `NetworkBehavior`. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Car** ; this will generate a NetworkObject class named **CarNetworkObject**.
```
Comment (yedpodtrzitko) [inline docs/mkdocs/docs/NetworkObject/changing-ownership.md]: also renamed to `NetworkBehavior`

**Comment by baflink** [comment]
👍

File: docs/mkdocs/docs/NetworkObject/changing-ownership.md (hunk 2 above)
Comment (baflink) [reply docs/mkdocs/docs/NetworkObject/changing-ownership.md]: Same reply as above

**Comment by baflink** [comment]
My comments were fixed by this commit:  https://github.com/BeardedManStudios/ForgeNetworkingRemastered/commit/fea13baea292d8d8ccb6886ed3c5b42c4fd09324

**Pull request merged**